client/src/components/dashboard/Poll.css

```diff
@@ -0,0 +1,82 @@
+.Poll-card {
```

Comment:
there doesn't seem to be a ClassName called Poll-card, I don't think this css selector is doing anything at the minute.

Reply:
fixed